support/cas-server-support-themes-collection/src/main/resources/templates/twbs/fragments/scripts.html

```diff
@@ -0,0 +1,14 @@
+<script type="text/javascript" th:src="@{${#themes.code('webjars.bootstrapmin.js')}}"></script>
+<script type="text/javascript" th:src="@{${#themes.code('cas.standard.js.file')}}"></script>
+
+<script th:inline="javascript">
+(function () {
+    document.addEventListener('DOMContentLoaded', function () {
```

Comment:
In the default template, this function gets called in scripts.html. Since this template is being overwritten, I needed to port that over. Since we aren't using cssVars I removed that section.

Reply:
Nvm, found the new references and removed scripts.html